webapp/app/js/compare.js

```diff
@@ -0,0 +1,494 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+
+// -------------------------------------------------------------------------
+// Utility Functions
+// -------------------------------------------------------------------------
+
+
+/**
+ * Compute the standard deviation for an array of values.
+ * 
+ * @param values
+ *        An array of numbers.
+ * @param avg
+ *        Average of the values.
+ * @return a number (the standard deviation)
+ */
+function stddev(values, avg) {
+  if (values.length <= 1) {
+    return 0;
+  }
+
+  return Math.sqrt(
+    values.map(function (v) { return Math.pow(v - avg, 2); })
+          .reduce(function (a, b) { return a + b; }) / (values.length - 1));
+}
+
+
+// -------------------------------------------------------------------------
+// End Utility Functions
+// -------------------------------------------------------------------------
+
+
+var comparetalos = angular.module("perf", ['ui.router', 'ui.bootstrap']);
+
+/* Copied from providers.js */
+comparetalos.provider('thServiceDomain', function() {
+    this.$get = function() {
+        if (window.thServiceDomain) {
+            return window.thServiceDomain;
+        } else {
+            return "";
+        }
+    };
+});
```

Comment:
This isn't needed now that bug 1152376 has landed. You can just pull this code out (it will be pulled in as part of the treeherder module).